interfaces/automated-testing/scd/design/operational-intent-format/uas_flight_auth_uas_serial_number.plantuml

```diff
@@ -0,0 +1,15 @@
+@startuml
+center header Operational Intent Format
+title UAS Flight Auth uas_serial_number
+
+participant "Test Driver" as TD
+participant "USS under test" as USS
+
+== Invalid ==
+TD --> USS: PUT /inject/operational_intent\nwith invalid uas_serial_number
```

Comment:
Operational intents don't have serial numbers; should this be a notional /inject/flight_authorisation endpoint rather than /inject/operational_intent?

Reply:
It makes sense and simplify the fact that this endpoint is optional for USS which doesn't need to comply with U-Space regulation.